cumulus_api/main.py

```diff
@@ -131,7 +134,7 @@ def update_provider(self, data):
         :return: message of success or raise error
         """
         record_type = f"providers/{data['id']}"
-        return self.__crud_records(record_type=record_type, verb="put", data=data)
+        return self.__crud_records(record_type=record_type, verb="patch", data=data)
```

Comment:
update provider still uses PUT: https://github.com/nasa/cumulus/blob/v16.0.0/packages/integration-tests/api/api.js#L334

Reply:
The wording in the Cumulus change log, as well as a Slack thread that I pulled up sounded very definitive that *all* uses of PUT should be transitioned to PATCH. However you are correct that Cumulus hasn't done that in v16 api-client. My interpretation was that they added identical support for both PUT and PATCH to allow for the transition until the PUT implementations actually start diverging. But that doesn't appear to be the case. I will update out branch and PR appropriately.

Reply:
Reverted.